doc/modules/feature_extraction.rst

```diff
@@ -100,6 +100,62 @@ of the time. So as to make the resulting data structure able to fit in
 memory the ``DictVectorizer`` class uses a ``scipy.sparse`` matrix by
 default instead of a ``numpy.ndarray``.
 
+.. _column_transformer:
+
+Columnar Data
+=============
+Many datasets contain features of different types, say text, floats and dates,
+where each type of feature requires separate preprocessing.
+Often it is easiest to preprocess data before applying scikit-learn methods, for example using
+pandas.
+If the preprocessing has parameters that you want to adjust within a
+grid-search, however, they need to be inside a transformer. This can be
```

Comment:
they=?

I'd remove the whole sentence and just say ":class:`ColumnTransformer` is a convenient way to perform heterogeneous preprocessing on data columns within a pipeline.